tests/testthat/test-benchmark-results.R

```diff
@@ -71,6 +71,17 @@ with_mock_dir(test_path("resp-class"), {
       "data.frame"
     )
   })
+
+  test_that("benchmark_results() respects the limit parameter", {
+    limit_test <- benchmark_results(run_reason = "test", limit = 2)
+    expect_true(nrow(limit_test) == 2L)
+  })
+
+  test_that("benchmark_results() respects the days parameter", {
+    day_test <- benchmark_results(run_reason = "nightly", days = 2)
+    ## this is 3 before we look back 2 days from today
+    expect_true(length(unique(as.Date(day_test$timestamp))) == 3L)
+  })
```

Comment:
For my understanding, `limit 02` = two latest entries / while `days 02`= (today + yesterday)

Reply:
Yeah `limit = 2` is the last two benchmark results. But `days = 2` is today + the previous two days.

Reply:
Gotcha, thanks